### ISSUE TITLE
Add a title attribute to describe the Hypothesis iframe

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/product-backlog#784]
This issue is part of the larger accessibility epic written up here https://github.com/hypothesis/product-backlog/issues/753

Problem: 
We failed in this accessibility check under the `Info and Relationships` section because we don't currently have a title attribute that describes the Hypothesis iframe. Adding this would provide a label for the frame so users can determine which frame to enter and explore in detail. For more information on this, see the W3C guidelines here: https://www.w3.org/TR/WCAG20-TECHS/H64.html

Acceptance Criteria: 
 Iframe titles
- Add a title attribute to describe the Hypothesis iframe. Example taken from W3C website in image below: 

<img width="755" alt="screen shot 2018-08-30 at 6 05 04 pm" src="https://user-images.githubusercontent.com/7463523/44884032-3f711600-ac7f-11e8-90b9-0e5662d6f469.png">

**Comment by ajpeddakotla** [comment]
| User Engagement | Building Community | Level of Effort |
|---|---|---|
| 3 | 1 | 3 |
